app/main.py

```diff
@@ -2,0 +1,10 @@
+def copy_file(command: str) -> None:
+    if len(command.split()) != 3:
+        raise ValueError("Command can take only 2 arguments")
+    cp_command, file, new_file = command.split()
+    if cp_command != "cp":
+        raise ValueError("Command not found")
+
+    if file != new_file:
+        with open(file, "r") as file_in, open(new_file, "w") as file_out:
+            file_out.write(file_in.read())
```

Comment:
You are using the same name `file` for the input file and the file object, which can lead to confusion. It would be better to use different names for these variables. For example, `input_file` and `output_file` for the file objects.